pkg/k8sutil/pod.go

```diff
@@ -112,3 +113,44 @@ func PodsHaveTheSameOwner(pods []corev1.Pod) bool {
 
 	return true
 }
+
+func SecurePodContext(user int64, group int64, isStrict bool) *corev1.PodSecurityContext {
```

Comment:
This function was moved from another file with a minor modification to accept a group as a parameter.